tools/chpl-language-server/chpl-shim

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 #
-# Copyright 2024-2024 Hewlett Packard Enterprise Development LP
+# Copyright 2024-2025-2025 Hewlett Packard Enterprise Development LP
```

Comment:
And this one.  Interestingly some of the newer tools ones are just fine despite probably also being added this year.